Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde_qs = "0.4.1"
 slog = "2.4.1"
 slog-term = "2.4.0"
 slog-async = "2.3.0"
+cookie="0.11"
```

Comment:
from what I understand, to use `secure` cookies, you need to enable that [feature](https://docs.rs/cookie/0.11.0/cookie/#features). Seems like that's something we'd want to include

Reply:
Yes that is true. But I wanted have the basic cookie extractor there before looking into secure cookie. Secure cookie needs `ring` which I believe is not a dependency yet.